include/ola/messaging/MessagePrinter.h

```diff
@@ -40,6 +40,7 @@ class MessagePrinter: public MessageVisitor {
 
     virtual void Visit(const BoolMessageField*) {}
     virtual void Visit(const IPV4MessageField*) {}
+    virtual void Visit(const IPV6MessageField*) {}
```

Comment:
Any reason to have "IPV6MessageField" instead of "IPv6MessageField"? I mean, sure, IPV4 has been here before but IPv4 and IPv6 are easier to read then IPV4 and IPV6

Reply:
Just legacy/compatibility for now really, we've generally gone for either CamelCase or being consistently upper-cased, and IPVx tracks through a lot of other code too, e.g. IPV4SocketAddress

I find DmxBuffer more jarring personally!

Also we should probably prioritise fixing this, which renders out user-facing code before worrying too much about our internal names:
https://github.com/OpenLightingProject/ola/blob/653449120b2d5630edddda123c480880fe324d4f/common/utils/StringUtils.cpp#L437-L453

Although at least C++ has it, the Python doesn't have anything from memory. Obviously ideally there would be one place, that could ideally handle the more complicated cases, for this to be managed. probably via a shell script that generates an include file or something.

Reply:
Sure, IPV4SocketAddress is fine for me (as is DmxBuffer :D). It was just for better readability since IPv4 and IPv6 are much more commonly used.